assets/src/components/vehiclePropertiesPanel.tsx

```diff
@@ -28,13 +28,19 @@ const VehiclePropertiesPanel = ({ selectedVehicle }: Props) => {
 &destination=${selectedVehicle.latitude.toString()},${selectedVehicle.longitude.toString()}\
 &travelmode=driving`
 
+  const titleText =
+    selectedVehicle.route_id +
+    (selectedVehicle.via_variant ? "-" + selectedVehicle.via_variant : "") +
```

Comment:
Some variants are named `_` (it tends to be the default), so I figured `225-_` would look better than `225__`. @ingridpierre is this okay?

Reply:
Honestly I've seen it notated several different ways in official documents.

- 57-1
- 57-_
- 57_1
- 571
- 57.1

I don't think it makes a big difference as long as we're consistent. I prefer the underscore personally because it's a little bit of formatting that makes it look different from a run number or block.

Reply:
Going with [this format](https://github.com/mbta/skate/pull/50#issuecomment-494501160)